### ISSUE TITLE
Operating Computer Ideas

### ISSUE THREAD

**Issue by Super3222** [yogstation13/yogstation-classic#847]
So let's refer to #10, which suggests that people who are laid out on the surgery table's implants should show up on a surgery computer around it. 

Implants like the syndicates own will show up as "#ERROR IMPLANT NOT RECOGNISED#" and whatever. 

That's a pretty good idea, and I was planning on working it at one point in time, but still far too easy.

So currently the operating computer's functions is to tell you the vitals of the patient on the surgery table as well as the next step in the surgery you're conducting (which literally no one checks for).

So that's why I wanted to suggest we add a big book in surgery.
One called "Surgery for dummies: a guide that teaches you more to surgery other than conducting sex changes".
It'd realistically, link to the wikia's page on surgery. And some information on the operation table.

So  operating tables and computers should be able to be researched in R&D. That's a given, and would most likely require a rollerbed upon construction.

But what I wanted to get at is more functions being given to the operating computer.
I want it to be able to detect different implants in people as well as different organs, creating a list of them.

However to foreign implants/organs, it won't be able to notice them.

I was thinking the operating computer could have a mode, where it switches to a page with two slots, that will either accept implants/organs. The computer will be be able to scan them, save it to the widespread database of organs that will be previously saved.

And when switching back to the patient mode, the computer would be able to notice the organ that is inside of the patient, and list it.

So, for these implants/organs to be scanned they will need to be placed in a container. Because you can't just slap a xenomorphs brain up against a computer and expect it to be downloaded instantly.

Implants already have their implant cases, so that's pretty handled already.
Organs on the other hand, will be tricky. The computer will not accept spoiled organs, so that means you will have to keep it within a freezer. But how would you be able to conceal it?

That's why I was thinking of an implementation of mini-coolers for medical doctors / surgeons, to conceal ONE organ at a time. (Though, it may have to come with a bag of ice inside of it, running processes of cooling down the objects inside of there.)

 And any other small item they may want to. It'd be a bulky item, like a toolbox, but less space.

The mini-cooler would run a similar process to the freezers, where they'd consistently keep organs at a low temperature preventing them from being spoiled.

The mini-cooler may actually not look like a cooler, but a cubicle like object.

**tldr**:

*operating computers list the organs/implants (that are in their database) inside of patients on the surgery table
*operating computers can scan organs/implants that aren't in their database
*mini-cooler implementation with a bag of ice to cool down organs inside of it.

**Comment by ShadowDeath6** [comment]
will this cause any shit with cruix's ling update? an instant "ling confirm" table doesn't seem like a good idea

**Comment by Super3222** [comment]
> will this cause any shit with cruix's ling update? an instant "ling confirm" table doesn't seem like a good idea

No it won't. With what I'm suggesting, they would need to make sure the chang organ doesn't decay, put it in a mini cooler, and shove it into the operating computer, scan it, remove it, and than any organ that is EXACTLY that type of changeling organ can be identified in the patients receiving surgery.
